curator-framework/src/main/java/org/apache/curator/framework/api/RemoveWatchesBuilder.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.curator.framework.api;

```

Comment:
Under what conditions can getZookeeperClient().getZookeeper() return null? Is it when we're trying to reconnect?

What should be done under these conditions? Presumably we just need to throw an exception back to the user because  we can't execute the command.